jupyter-docker/Dockerfile

```diff
@@ -0,0 +1,120 @@
+# adapted from https://hub.docker.com/r/jupyter/base-notebook/ AKA https://github.com/jupyter/docker-stacks/tree/master/base-notebook
+
+# Debian Jessie debootstrap from 2017-02-27
+# https://github.com/docker-library/official-images/commit/aa5973d0c918c70c035ec0746b8acaec3a4d7777
+FROM debian@sha256:52af198afd8c264f1035206ca66a5c48e602afb32dc912ebf9e9478134601ec4
+
+USER root
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBIAN_REPO http://cdn-fastly.deb.debian.org
+
+ENV JAVA_VER jdk1.8.0_131
+ENV JAVA_TGZ jdk-8u131-linux-x64.tar.gz
+ENV JAVA_URL http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/$JAVA_TGZ
+ENV JAVA_HOME /usr/lib/jdk/$JAVA_VER
+
+ENV SPARK_VER 2.0.2
+ENV SPARK_TGZ spark-$SPARK_VER.tgz
+ENV SPARK_URL http://d3kbcqa49mib13.cloudfront.net/$SPARK_TGZ
+ENV SPARK_HOME /spark/spark-$SPARK_VER
+
+# result of `gsutil cat gs://hail-common/latest-hash.txt` on 6 June 2017
+ENV HAILHASH 53e9d33
```

Comment:
Q: as the dev team responsible for maintaining this dependency, how do we do a good job of keeping this up to date and knowing it needs bumping?